### DISCUSSION TITLE
Freebsd Support

### DISCUSSION THREAD

**Discussion by jeezs** [tauri-apps/tauri#5117] [Q&A]
Is it possible to create Freebsd compatible app with Taury?
If not, how can I proceed?, what is the effort to make compatible ?
Thanks

**Comment by kellytk** [comment]
Any update on this? Official FreeBSD support would add another advantage over Electron.

**Comment by clean-master** [comment]
i really need this

**Comment by jeezs** [comment]
﻿Me too!!!!Envoyé de mon iPhoneEnvoyé de mon iPhoneLe 19 juil. 2023 à 06:59, 清理大师 ***@***.***> a écrit :﻿
i really need this
